src/api/v1/db/oracledb/doughs-dao.js

```diff
@@ -114,12 +114,35 @@ const doughsGetQuery = (conditionals) => `SELECT ${doughColumnAliases} FROM DOUG
 
 /**
  * A query to create a new dough
+ *
  * @constant
  * @type {string}
  */
 const doughsPostQuery = `INSERT INTO DOUGHS (${doughColumns}) VALUES (${doughValues}) 
```

Comment:
Just found there's a trailing space at the end of this line, also at line 141.

Reply:
👍 adding this as a rule in `.eslintrc`